src/js/components/Form/Form.js

```diff
@@ -289,6 +299,12 @@ const Form = forwardRef(
           return result;
         };
 
+        if (required) {
+          setRequiredFields(prevValue =>
+            !prevValue.includes(name) ? [...prevValue, name] : [...prevValue],
```

Comment:
I think we can just return `prevValue` directly in the false case, don't need to create a new array.

Reply:
missed it.